Source/Widgets/Viewer/Viewer.js

```diff
@@ -1855,19 +1872,56 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
     };
 
     function updateZoomTarget(viewer) {
-        var entities = viewer._zoomTarget;
-        if (!defined(entities) || viewer.scene.mode === SceneMode.MORPHING) {
+        var target = viewer._zoomTarget;
+        if (!defined(target) || viewer.scene.mode === SceneMode.MORPHING) {
             return;
         }
 
         var scene = viewer.scene;
         var camera = scene.camera;
         var zoomPromise = viewer._zoomPromise;
         var zoomOptions = defaultValue(viewer._zoomOptions, {});
+        var options;
+
+        // If zoomTarget was Cesium3DTileset
+        if (defined(target.readyPromise)) {
```

Comment:
`instanceof` should work properly now since you fixed the error with the import.